jquery.pnotify/index.d.ts

```diff
@@ -257,6 +257,36 @@ interface PNotifyOptions {
     labels?: PNotifyLabel;
 }
 
+interface PNotifyStyling {
+    container?: string,
+    notice?: string,
+    notice_icon?: string,
+    info?: string,
+    info_icon?: string,
+    success?: string,
+    success_icon?: string,
+    error?: string,
+    error_icon?: string,
+    // buttons
+    closer?: string,
```

Comment:
I was able to find everything above `// buttons`, but could you point out where this and the below properties are used int the original source?